wasm/wazeroir/interpreter.go

```diff
@@ -70,10 +70,8 @@ func (it *interpreter) drop(r *InclusiveRange) {
 }
 
 func (it *interpreter) pushFrame(frame *interpreterFrame) {
-	if buildoptions.CheckCallStackOverflow {
-		if buildoptions.CallStackHeightLimit <= len(it.frames)-1 {
-			panic(wasm.ErrCallStackOverflow)
-		}
+	if buildoptions.CallStackHeightLimit <= len(it.frames)-1 {
```

Comment:
While there let's reduce the cost of each operation😄
```suggestion
	if buildoptions.CallStackHeightLimit <= len(it.frames) {
```